server/tests/test_sigma_canary.py

```
@@ -0,0 +1,123 @@
+    def test_benign_commands(self, sigma_rules, cmd):
+        v = _check(sigma_rules, cmd)
+        assert not v.matched, (
+            f"False positive on benign command: {cmd!r} "
+            f"-> {v.rule_id}: {v.description}"
+        )
+
+
+class TestSigmaRulesDetectMalicious:
+    """Commands that Sigma rules should catch (beyond hand-written rules)."""
+
+    @pytest.mark.parametrize("cmd,expected_prefix", [
```

Ruff (0.15.11)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)